cmd/node/main.go

```diff
@@ -377,15 +376,22 @@ func main() {
 			if globalCfg.PprofPort == 0 {
 				globalCfg.PprofPort = int(time.Now().Unix()%100) + 61000
 			}
-			ln, err := net.Listen("tcp", fmt.Sprintf("0.0.0.0:%d", globalCfg.PprofPort))
-			if err != nil {
+
+			server := &http.Server{
```

Comment:
This is the pprof endpoint for debuging purposes. We don't want to have a timeout of 10 seconds, some times it requires more than 30 minutes waiting time in order to recover a big enough trace for analyzing the cpu, memory, etc... profiles. 

This endpoint is only enabled in dev scenarios, so no need to protect it against potential slowloris attacks.